templates/horizon/config/httpd.conf

```
@@ -22,8 +22,8 @@ LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combine
 LogFormat "%{X-Forwarded-For}i %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" proxy
 
 SetEnvIf X-Forwarded-For "^.*\..*\..*\..*" forwarded
-CustomLog /dev/stdout combined env=!forwarded
-CustomLog /dev/stdout proxy env=forwarded
+CustomLog /var/log/horizon/horizon.log combined env=!forwarded
+CustomLog /var/log/horizon/horizon.log proxy env=forwarded
```

Let's use the `LogFile` const in this template actually. That will be cleaner:
```suggestion
CustomLog {{ .LogFile }} combined env=!forwarded
CustomLog {{ .LogFile }} proxy env=forwarded
```

---

Note that you're already using that in this file:
https://github.com/openstack-k8s-operators/horizon-operator/pull/399/files#diff-7daba0607a376b7191dae912ac01337fb3debaf18495b1e15e0c4a940ac6c199R2